src/lockfile/index.js

```diff
@@ -102,14 +102,14 @@ export default class Lockfile {
 
   static async fromDirectory(dir: string, reporter?: Reporter): Promise<Lockfile> {
     // read the manifest in this directory
-    const lockfileLoc = path.join(dir, constants.LOCKFILE_FILENAME);
+    const lockfileLoc = path.join(dir, LOCKFILE_FILENAME);
 
     let lockfile;
     let rawLockfile = '';
     let parseResult;
 
-    if (await fs.exists(lockfileLoc)) {
-      rawLockfile = await fs.readFile(lockfileLoc);
+    if (await exists(lockfileLoc)) {
```

Comment:
Using

`import {exists, readFile} from './util/fs.js'`, `exist()` and `readFile`

instead of

`import fs from './util/fs.js'`, `fs.exist()` and `fs.readFile()`

appears to have no effect on the bundle output size. Huh. I was probably just being overly careful. I can return to the olden way.